components/isceobj/RtcProc/runNormalize.py

```diff
@@ -18,25 +20,29 @@ def runNormalize(self):
     '''
     Make sure that a DEM is available for processing the given data.
     '''
-   
     refPol = self._grd.polarizations[0]
     master = self._grd.loadProduct( os.path.join(self._grd.outputFolder, 'beta_{0}.xml'.format(refPol)))
 
 
     azlooks, rglooks = self._grd.getLooks( self.posting, master.groundRangePixelSize, master.azimuthPixelSize, self.numberAzimuthLooks, self.numberRangeLooks)
 
 
-    if (azlooks == 1) and (rglooks == 1):
-        return
-
-    slantRange = False
     for pol in self._grd.polarizations:
-        inname = os.path.join( self._grd.outputFolder, 'beta_{0}.img'.format(pol) )
-        takeLooks(inname, azlooks, rglooks)
-
-        if not slantRange:
-            inname = master.slantRangeImage.filename
-            takeLooks(inname, azlooks, rglooks)
-            slantRange = True
-
+        if (azlooks == 1) and (rglooks == 1):
+            inname = os.path.join( self._grd.outputFolder, 'beta_{0}.img'.format(pol))
+        else:
+            inname = os.path.join( self._grd.outputFolder, filenameWithLooks('beta_{0}.img'.format(pol), azlooks, rglooks))
+
+        incname = os.path.join(self._grd.geometryFolder, self._grd.incFileName)
+        outname = os.path.join(self._grd.outputFolder, filenameWithLooks('gamma_{0}'.format(pol)+'.img', azlooks, rglooks))
+        maskname = os.path.join(self._grd.geometryFolder, self._grd.slMaskFileName)
+
+        cmd = "imageMath.py --e='a*cos(b_0*PI/180.)/cos(b_1*PI/180.) * (c==0)' --a={beta} --b={inc} --c={mask} -o {out} -t FLOAT -s BIL"
+
+        cmdrun = cmd.format(inc = incname,
+                            beta = inname,
+                            out = outname,
+                            mask = maskname)
+        status = os.system(cmdrun)
```

Comment:
Would it be possible to replace the os.system() command with direct calls to imageMath.main().
We are trying to not use system calls within the run methods. If not, we can add that as a separate PR. 

For separating the args and files:

```
class Dummy:
    pass

args = Dummy()
args.equation = 'a*cos(b_0*PI/180.)/cos(b_1*PI/180.) * (c==0)'
args.dtype = np.float32
args.scheme = 'BIL'
args.out = outname


files  = Dummy()
files.a = inname
files.b = incname
files.c =  maskname

imageMath.main(files, args)
```

Reply:
Thanks Piyush! That's no problem, I will look into that soon ... thanks for letting me know.

Reply:
Hi Piyush, I just updated the codes to avoid the system call as you suggested. I also slightly modified imageMath.py because I encountered some issues otherwise. Thank you!